erpnext/templates/pages/help.py

```diff
@@ -5,22 +5,74 @@
 
 def get_context(context):
 	context.no_cache = 1
+	context.bg = 'background-color: #fafbfc; border-radius:0'
```

Comment:
Not a good practice

Reply:
Was trying to minimize code to use this variable in HTML